Linux_cmake/nheqminer_cpu/CMakeLists.txt

```diff
@@ -0,0 +1,137 @@
+project(nheqminer_cpu)
+cmake_minimum_required(VERSION 2.8)
+
+#aux_source_directory(. SRC_LIST)
+#add_executable(${PROJECT_NAME} ${SRC_LIST})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+# LINUX
+if(CMAKE_COMPILER_IS_GNUCXX)
+    # use native cpu features
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=native")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native")
+    # optimizations
+    add_definitions(-O3)
```

Comment:
> trying this mod now (also added -march=core-avx2 to see if it matters) 

compiles & runs fine for me w/ `Ofast`. 
Btw, unless you try to be portable, `march=native` is better than any other (auto picks all avail optimizations for your HW)